oesukam/andela-sendit
# #162036881 Add Parcel journey distance to update a parcel page

#### What does this PR do?
Adds distance matrix to update a parcel page

#### Description of Task to be completed?
- Add distance matrix to update a parcel page

#### How should this be manually tested?
- Login into your account
- Navigate my profile
- Click on edit parcel if exist or add a new parcel
- Check journey distance from the map

#### What are the relevant pivotal tracker stories?
`#162036881`
